conf/httpd.conf.d/httpd.graphite

```diff
@@ -101,10 +101,10 @@ $KeepAliveTimeout = "10";
 $KeepAlive = "On";
 $MaxKeepAliveRequests = "100";
 
-$MaxClients = "56";
-$StartServers = "4";
-$MinSpareServers = "4";
-$MaxSpareServers = "20";
+$MaxClients = "2";
+$StartServers = "1";
+$MinSpareServers = "1";
+$MaxSpareServers = "1";
```

Comment:
Yes but they would share the same pool so it could reduce the amount of processes that they're using even though each process would use a bit more memory